xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -749,13 +751,45 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
 
     CLog::Log(LOGINFO, "CDVDVideoCodecAndroidMediaCodec::Open Testing codec: {}", m_codecname);
 
+    // There is some confusion for video/VC1 vs. video/wvc1 especially on Sony devices, while
+    // IANA defines VC-1 as video/vc1, Android API defines it as video/wvc1 - be nice and test it
+    bool isVC1 = StringUtils::Contains(m_codecname, "vc1", false);
```

Comment:
```suggestion
    const bool isVC1 = m_codecname.find("vc1") != std::string::npos;
```
Seems is equivalent functionality and avoids include "StringUtils.h" only for use here

Reply:
I wanted to be "Case-Insensitive" if a Decoder would be called: myname.VC1.whatever

Reply:
```suggestion
    const bool isVC1 = StringUtils::Contains(m_codecname, "vc1", true);
```

```
bool StringUtils::Contains(std::string_view str,
                           std::string_view keyword,
                           bool isCaseInsensitive /* = true */)
```

Reply:
Oh man - that documentation, when one just reads over it and codes it blindly ...